awssqs/samples/display-resources.yaml

```diff
@@ -1,26 +1,25 @@
 # Channel for testing events.
 
-apiVersion: eventing.knative.dev/v1alpha1
+apiVersion: messaging.knative.dev/v1alpha1
 kind: Channel
```

Comment:
why not directly using the `InMemeoryChannel` kind here ?

Reply:
For no real reason. Is there a benefit of using it directly? Or is there a knative preference of using the concrete CRD directly instead of the abstract channel? We can change it if you want, but then let's do it for all channels in `eventing-contrib` for the sake of consistency. What do you think?

Reply:
generic channel is OK - I was just wondering, why